packages/native/src/components/TextArea.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  COMPONENT_TYPES,
+  FIELD_NAME,
+  FORM_TYPES,
+  GROUPS,
+  PROP_TYPES,
+  TEXT_INPUT_PROPS,
+} from "@draftbit/types";
+import * as React from "react";
+import { TextInput, StyleSheet } from "react-native";
+
+const TextArea: React.FC<any> = ({ numberOfLines = 4, style, ...props }) => {
+  return (
+    <TextInput
+      {...props}
+      style={[styles.input, style]}
+      numberOfLines={numberOfLines}
+    />
+  );
+};
+
+const styles = StyleSheet.create({
+  input: {
+    borderWidth: 1,
+    padding: 10,
+  },
+});
+
+export const SEED_DATA = {
+  name: "Text Area",
+  tag: "TextArea",
+  description: "Render Text Area",
+  category: COMPONENT_TYPES.basic,
+  layout: { flex: 1 },
+  props: {
+    ...TEXT_INPUT_PROPS,
+    multiline: {
+      label: "Multiline",
+      description: "Multiline",
+      group: GROUPS.uncategorized,
+      formType: FORM_TYPES.boolean,
+      propType: PROP_TYPES.BOOLEAN,
+      defaultValue: true,
+      editable: true,
+      required: false,
+    },
+    numberOfLines: {
+      label: "Number Of Lines",
+      description: "Number Of Lines",
+      group: GROUPS.basic,
+      formType: FORM_TYPES.boolean,
+      propType: PROP_TYPES.BOOLEAN,
+      defaultValue: 4,
+      editable: true,
+      required: false,
+    },
+    fieldName: {
+      ...FIELD_NAME,
+      defaultValue: "textInputValue",
+      handlerPropName: "onChangeText",
+    },
+  },
+};
```

Comment:
Rather than creating a new component within draftbit/ui called TextArea, do you think it would be possible to just take this seed data that you created, and put it in a new file at `core/src/mappings/TextArea.js`? That way, you don't need to create a new component, and can re-use the TextInput under the hood.

In other words, a user adding a "Text Area" component will really just be adding a TextInput component, but with `multiline` set to `true` and `numberOfLines` set to `4` by default.

Reply:
Besides moving the seed data to that new file and deleting this file, the one other change you would need to make is making the tag for the component `TextInput` (so that it uses the TextInput component).